### PR TITLE
adding ucsc genome browser track creation script to pipeline

### PR DESCRIPTION
**Pull request by maximilianh**
This is a simple ~150 lines script that creates a UCSC track from the brcaexchange tarball. The readme file has all the commands to create the track. 

Once your track is created, you can paste the URL into the UCSC Genome Browser custom track box for testing and can reference it from any UCSC track hub. 

Custom track box: http://genome.ucsc.edu/cgi-bin/hgCustom

How to create a track hub text file: https://genome.ucsc.edu/goldenpath/help/hgTrackHubHelp.html

### CODE REVIEW

File: pipeline/genomeBrowserTrack/brcaToBed.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+for line in ifh:
+    row = line.rstrip("\n").rstrip("\r").split("\t")
+    rec = rowRec(*row)
+    rd = OrderedDict(zip(headers, row)) # row as dict
+
+    chrom = "chr"+rec.Chr
+    start = str(int(rec.pyhgvs_Hg37_Start)-1)
+    end = rec.pyhgvs_Hg37_End
+
+    #name = rec.Protein_Change
```
Comment (zfisch) [inline pipeline/genomeBrowserTrack/brcaToBed.py]: are these comments needed or can they be removed?

File: pipeline/genomeBrowserTrack/brcaExchange.as [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    lstring pyhgvs_Genomic_Coordinate_36; "pyhgvs Genomic Coordinate 36"
+    lstring pyhgvs_Genomic_Coordinate_37; "pyhgvs Genomic Coordinate 37"
+    lstring pyhgvs_Genomic_Coordinate_38; "pyhgvs Genomic Coordinate 38"
+    lstring pyhgvs_Hg36_Start; "pyhgvs Hg36 Start"
+    lstring pyhgvs_Hg36_End; "pyhgvs Hg36 End"
+    lstring pyhgvs_cDNA; "pyhgvs cDNA"
+    lstring pyhgvs_Protein; "pyhgvs Protein"
+    lstring change_type; "change type"
+    string _mouseOver; "mouse over field hidden"
+    )
```
Comment (zfisch) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: needs newline

File: pipeline/genomeBrowserTrack/brcaExchange.as [hunk 3]
```diff
@@ -0,0 +1,142 @@
+table brcaExchanges
+" bed9 + many additional fields from brcaExchange.org "
+    (
+    string chrom;      "Chromosome (or contig, scaffold, etc.)"
```
Comment (zfisch) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: it's likely that fields will change in the future -- is there a way to make this file update accordingly or will manual changes be required here?

**Comment by maximilianh** [comment]
Ok to remove

**Comment by zfisch** [comment]
@maximilianh did you see the other questions as well and could you please update the PR accordingly before merge?

File: pipeline/genomeBrowserTrack/brcaExchange.as (hunk 2 above)
Comment (maximilianh) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: there is no newline missing here. No idea what you mean.

File: pipeline/genomeBrowserTrack/brcaExchange.as (hunk 3 above)
Comment (maximilianh) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: no the chrom field will never change.

File: pipeline/genomeBrowserTrack/brcaToBed.py (hunk 1 above)
Comment (maximilianh) [inline pipeline/genomeBrowserTrack/brcaToBed.py]: Fell free to remove any comments or change as you see fit.

File: pipeline/genomeBrowserTrack/brcaExchange.as (hunk 3 above)
Comment (zfisch) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: im referring to the fields in general, not the chrom field specifically. For example, new fields may be added or field names may change -- in a recent change "Minor_allele_frequency_ESP_percent" was changed to "Minor_allele_frequency_percent_ESP"

File: pipeline/genomeBrowserTrack/brcaExchange.as (hunk 2 above)
Comment (zfisch) [inline pipeline/genomeBrowserTrack/brcaExchange.as]: screen shot 2017-07-27 at 10 44 38 am
The red circle is Github complaining that there's no newline at the end of the file. please add one.

**Comment by maximilianh** [comment]
There is no need for a new line here. Feel free to add one if you like new
lines.

Sorry I have no time. I have >30 emails a day and did everything you need
for this feature. New lines at EOF make no difference.

**Pull request merged**